pkg/kubectl/cmd/attach/attach.go

```diff
@@ -61,8 +61,8 @@ const (
 	defaultPodLogsTimeout   = 20 * time.Second
 )
 
-// AttachOptions declare the arguments accepted by the Attach command
-type AttachOptions struct {
```

Comment:
please leave this naming as before.

Reply:
Sounds good, I'll make a PR for other files instead